docs/guides/advanced/01_initialization.md

```diff
@@ -100,24 +100,23 @@ But you should see some initial blocks in postgres:
 select * from miniblocks;
 ```
 
-#### Our L1 (geth)
+#### Our L1 (reth)
 
 Let's finish this article, by taking a look at our L1:
 
-```shell
-docker container exec -it zksync-era-geth-1  geth attach http://localhost:8545
-```
-
-The command above will start a shell - and you can check that you're a (localnet) crypto trillionaire, by running:
+We will use the `web3` tool to communicate with the L1, have a look at [02_deposits.md](02_deposits.md) for installation
+instructions. You can check that you're a (localnet) crypto trillionaire, by running:
 
 ```shell
-eth.getBalance(personal.listAccounts[0])
+./web3 --rpc-url http://localhost:8545 balance 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
```

Comment:
what is `./web3`?

Reply:
it's a tool we use at another guide, I just reused it here

> We will use the `web3` tool to communicate with the L1, have a look at [02_deposits.md](02_deposits.md) for installation
instructions.